release notes/v0.50.0.md

```diff
@@ -6,8 +6,16 @@ k6 `v0.50.0` is here 🎉! This release includes:
 
 ## Breaking changes
 
-- `#pr`, `<small_break_1>`
-- `#pr`, `<small_break_2>`
+### Browser APIs to Async
+
+In future release we are going to be moving most of the browser APIs to asynchronous (promisfying them). We expect that this will affect majority of our users, which is why we are posting this upfront before we make the change itself. Here are the reasons for making this large breaking change:
+
+1. Most of the browser APIs use some form of long running IO operation (networking) to perform the requested action on the web browser against the website under test. We need to avoid blocking the javascript's runtime event loop for such operations.
```

Comment:
```suggestion
1. Most browser APIs use some form of long-running IO operation (networking) to perform the requested action on the web browser against the website under test. We need to avoid blocking Javascript's runtime event loop for such operations.
```